clang/test/dpct/query_api_mapping/Thrust/thrust_api_test.cu

```diff
@@ -0,0 +1,262 @@
+// UNSUPPORTED: cuda-8.0
```

Comment:
seem like only 23 API lit test

Reply:
lit test for stable_partition is missing, will be added in the updated PR.